### PR TITLE
fix #8783: incorrect icon for monitored file in tab bar dialog and vertical dir switch

### LINKED ISSUE #8783: Monitored file has incorrect icon in Ctrl+Tab switcher
### Steps to Reproduce the Issue (STR)
1. Have several tabs open in N++
2. For a disk file, turn monitoring on (press the "eye" icon in the toolbar).
3. Cycle through the files using Ctrl+Tab repeatedly.

### Expected Behavior

I expected to see the "eye" icon like the one used on the tab bar, for the monitored file.

### Actual Behavior

Icon used for the monitored file seems to be the "readonly" icon (grey diskette).

image

### Debug Information

Notepad++ v7.8.8   (64-bit)
Build time : Jun 23 2020 - 23:59:56
Path : C:\.........\npp.7.8.8.bin.x64\notepad++.exe
Admin mode : OFF
Local Conf mode : ON
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 1809
OS Build : 17763.1339
Current ANSI codepage : 1252
Plugins : mimeTools.dll NppConverter.dll NppExport.dll 

### Additional Information

For color comparison, here's a cap of a non-monitored file (note the blue icon -- contrast with the grey icon in the earlier cap):

image

### PR DESCRIPTION
**Pull request by ErwanDelhoveUCL**
Fixes #8783

Small fixed of 8783
Awaiting comments! ^^

### CODE REVIEW

File: PowerEditor/src/WinControls/VerticalFileSwitcher/VerticalFileSwitcherListView.cpp [hunk 1]
```diff
@@ -222,7 +222,7 @@ void VerticalFileSwitcherListView::setItemIconStatus(BufferID bufferID)
 	LVITEM item;
 	item.pszText = fn;
 	item.iSubItem = 0;
-	item.iImage = buf->getUserReadOnly()||buf->getFileReadOnly()?2:(buf->isDirty()?1:0);
+	item.iImage = buf->getUserReadOnly()?3:buf->getFileReadOnly()?2:(buf->isDirty()?1:0);
```
Comment (donho) [inline PowerEditor/src/WinControls/VerticalFileSwitcher/VerticalFileSwitcherListView.cpp:225]: `(buf->getUserReadOnly() || buf->getFileReadOnly())` -> Both are read only, so it's the same icon.
You need use another way to detect monitoring status.

File: PowerEditor/src/Notepad_plus.cpp [hunk 2]
```diff
@@ -53,7 +53,7 @@
 
 using namespace std;
 
-enum tb_stat {tb_saved, tb_unsaved, tb_ro};
+enum tb_stat {tb_saved, tb_unsaved, tb_ro,tb_monitored};
```
Comment (donho) [inline PowerEditor/src/Notepad_plus.cpp:56]: Add a while space in front of tb_monitored

File: PowerEditor/src/WinControls/VerticalFileSwitcher/VerticalFileSwitcherListView.cpp (hunk 1 above)
Comment (ErwanDelhoveUCL) [reply PowerEditor/src/WinControls/VerticalFileSwitcher/VerticalFileSwitcherListView.cpp:225]: Do you think I should replace the int number with an enumeration or it isn't useful here?
I will make a method isReadOnly() like in the Notepad_plus.cpp.

Comment (ErwanDelhoveUCL) [reply PowerEditor/src/WinControls/VerticalFileSwitcher/VerticalFileSwitcherListView.cpp:225]: My bad the method isReadOnly() is already there ^^

Comment (donho) [reply PowerEditor/src/WinControls/VerticalFileSwitcher/VerticalFileSwitcherListView.cpp:225]: @ErwanDelhoveUCL

After applying your PR, when I set document to ReadOnly, the icon become monitoring:

image

**Review by donho — CHANGES_REQUESTED** [review]
Keep  `buf->getUserReadOnly()||buf->getFileReadOnly()` for 2 and use `isMonitoringOn()` for detecting 3

**Review by donho — APPROVED** [review]

**Pull request closed without merging**